fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -24,6 +24,8 @@ fi
 
 source $HCPPIPEDIR/global/scripts/log.shlib  # Logging related functions
 source $HCPPIPEDIR/global/scripts/opts.shlib # Command line option functions
+source ${HCPPIPEDIR}/global/scripts/processingmodecheck.shlib  # Check processing mode requirements
```

Comment:
Can we put $HCPPIPDIR in brackets for the preceding two lines for consistency.

Reply:
That is already a part of the debug PR, but sure. I changed it.